lib/remote/httpserverconnection.hpp

```diff
@@ -28,7 +28,7 @@ class HttpServerConnection final : public Object
 	HttpServerConnection(const String& identity, bool authenticated, const Shared<AsioTlsStream>::Ptr& stream);
 
 	void Start();
-	void Disconnect();
```

Comment:
That method can now be made private and a quick doc comment for it stating that it must be called from a coroutine running on `m_IoStrand` would be very nice.